app/main.py

```diff
@@ -1,3 +1,50 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __eq__(self, other):
+        return self.km == other
+
+    def __ne__(self, other):
+        return self.km != other
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if type(other) is not Distance:
+            return Distance(self.km + other)
+        else:
+            return Distance(self.km + other.km)
+
+    def __iadd__(self, other):
+        if type(other) is not Distance:
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round(self.km / other, 2))
+
+    def __gt__(self, other):
+        return self.km > other
+
+    def __lt__(self, other):
+        return self.km < other
+
+    def __ge__(self, other):
+        return self.km >= other
```

Comment:
here you can use magic method you wrote (don't forget to use it as a magic method)
ge == not lt